### PR TITLE
Clean up homepage.

### PR DESCRIPTION
**Pull request by vitillo**
I cleaned up the layout, removed references to telemetry v2 and added links to the other *.t.m.o. sites.

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/telemetry-dashboard/272)

### CODE REVIEW

**Comment by vitillo** [comment]
@chutten r?

File: index.html [hunk 1]
```diff
@@ -17,58 +17,36 @@
...
-                        <div class="dashboard-description">View evolution of measure values over time.</div>
+                    <a class="list-group-item" href="https://analysis.telemetry.mozilla.org/">
+                        Analysis Portal <div class="dashboard-description">Analyze telemetry data with Spark.</div>
                     </a>
-                    <a class="list-group-item" href="/advanced/">
-                        <span class="label label-default pipeline-label">Telemetry v2</span>
-                        Old Dashboard
-                        <div class="dashboard-description">View and compare distributions and evolutions.</div>
+                    <a class="list-group-item" href="https://reports.telemetry.mozilla.org/">
+                        Reports Portal <div class="dashboard-description">Explore data analyses & reports.</div>
```
Comment (chutten) [inline index.html]: Need an HTML entity here, no?

**Comment by chutten** [comment]
Is there a live preview I can look at?

**Comment by vitillo** [comment]
There isn't but it's simple enough to try it out locally :)

**Comment by chutten** [comment]
LGTM

**Pull request merged**